src/api/volume.go

```diff
@@ -100,6 +100,10 @@ func (s *VolumeService) Attach(ctx context.Context, volume *csi.Volume, server *
 		if hcloud.IsError(err, hcloud.ErrorCode("limit_exceeded_error")) {
 			return volumes.ErrAttachLimitReached
 		}
+		//TODO: update once hcloud upgraded to v1.12
```

Comment:
I think this part can be included in a PR about fixing the locked server errors CSI-driver receives from the hcloud.